tests/publiccloud/check_services.pm

```diff
@@ -21,12 +21,12 @@ sub run {
 
     my $instance = $args->{my_instance};
 
-    # Debug
-    $instance->ssh_script_run('systemctl --no-pager list-units');
-
     # Check if there are any failed services
     # 1. Get the list of failed services from the SUT
-    my $failed_services_output = $instance->ssh_script_output('sudo systemctl --failed --no-legend --plain | awk "{print \\$1}"');
+    my $failed_services_output = $instance->ssh_script_output(
+        'sudo systemctl --failed --no-legend --plain | awk "{print \\$1}"'
```

Comment:
`--no-pager` should disable the paging output of systemctl, otherwise the awk reads from `less` or whatever pager is set on the SUT